src/Components/MyRockets/RocketItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { ReserveRocket } from '../../redux/rockets/Rocket';
+
+const RocketItem = (props) => {
+  const {
+    id, name, description, flickrImage, reserved,
+  } = props;
+
+  const dispatch = useDispatch();
+
+  const onClick = () => {
+    dispatch(ReserveRocket(id));
+  };
+
+  return (
+    <div>
+      <li className="rocketitem" id="flex">
+        <img src={flickrImage} alt="Girl in a jacket" width="250" height="250" />
+        <div className="rocketright" id="flex">
+          <h2>{name}</h2>
+          <div className="spantext" id="flex">
+            <span className={reserved ? 'active' : 'inactive'}>
+              reserved
```

Comment:
- [ ] The parent element of each `<RocketItem />` is a `<ul>`, but the items are contained inside a `<div>`, ending up with this markup structure:

![image](https://user-images.githubusercontent.com/77069011/217967244-82fdd3e7-cc67-4caa-b5d8-e819fb742b8b.png)

Please consider removing the `<div>` and keeping the `<li>` as the top-level container of each `<RocketItem />` as it is the correct markup syntax! 🤓 🔧